packages/@aws-cdk/aws-codepipeline-actions/test/codestar-connections/codestar-connections-source-action.test.ts

```diff
@@ -148,27 +148,155 @@ describe('CodeStar Connections source Action', () => {
       ],
     });
 
+  });
+
+  test('exposes variables', () => {
+    const stack = new Stack();
+
+    createBitBucketAndCodeBuildPipeline(stack, {
+      triggerOnPush: false,
+    });
+
+    expect(stack).toHaveResourceLike('AWS::CodePipeline::Pipeline', {
+      'Stages': [
+        {
+          'Name': 'Source',
+        },
+        {
+          'Name': 'Build',
+          'Actions': [
+            {
+              'Name': 'CodeBuild',
+              'Configuration': {
+                'EnvironmentVariables': '[{"name":"CommitId","type":"PLAINTEXT","value":"#{Source_BitBucket_NS.CommitId}"}]',
+              },
+            },
+          ],
+        },
+      ],
+    });
 
   });
+
+  test('exposes variables with custom namespace', () => {
+    const stack = new Stack();
+
+    createBitBucketAndCodeBuildPipeline(stack, {
+      triggerOnPush: false,
```

Comment:
Same question here (why are we passing `triggerOnPush` in this test?).

Reply:
Ditto